### PR TITLE
Create a desktop version for the personal portfolio using media queries

### PR DESCRIPTION
**Pull request by keebot-png**
List of actions that have taken place:

Set up and installed the correct files.
Updated Readme file in line with recent project requirements
Passed the linter errors
Implemented media queries to support desktop version at 768 pixels or wider.
Implemented animations and transitions in CSS which have hover effects over the menu icons, project buttons and the “Get in Touch” button.
Different Grid and flexbox methods were used throughout the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,51 +8,58 @@
...
-        <header class="toolbar-container">
-            <div class="name-container">
-                <a href="#" class="logo">Keenan</a>
-            </div>
-            <div class="menu-container">
-                <a href="#" class="menu"><img src="./Images/Union.svg" alt="Menu icon"></a>
-            </div>
-        </header>
+        <nav class="toolbar-container">
+            <!-- <div class="container-wrapper"> -->
```
Comment (awais-amjed) [inline index.html]: Kindly remove the commented code 😃

File: index.html [hunk 2]
```diff
@@ -8,51 +8,58 @@
...
+                </div>
+                <div class="menu-container">
+                    <a href="#" class="menu"><img src="./Images/Union.svg" alt="Menu icon"></a>
+                </div>
+            <!-- </div> -->
+            <!-- <div class="menu">
+                <a href="#" class="menu-item-1">Portfolio</a>
+                <a href="#" class="menu-item-2">About</a>
+                <a href="#" class="menu-item-3">Contact</a>
+            </div> -->
```
Comment (awais-amjed) [inline index.html]: Kindly remove the commented code 😃

File: index.html [hunk 3]
```diff
@@ -8,51 +8,58 @@
     <title>Keenan Botes personal portfolio </title>
 </head>
 <body>
-    <div class="container">
+    <main class="container">
```
Comment (awais-amjed) [inline index.html:11]: Kindly fix the extra space around the page as shown below 🤗

Your design:

image

Figma design:

image

Comment (awais-amjed) [inline index.html:11]: Kindly also note that the mobile design should also resemble the Figma design 🤗
- _Note the spaacing around the page_
- _The color of the sub-heading is different on mobile and desktop_

Your design:

image

Figma design:

image

File: index.html [hunk 4]
```diff
@@ -8,51 +8,58 @@
...
+                <li class="social-media-links"><a href="#" class="Medium-profile"><img src="./Images/medium.svg" alt="Medium logo" class="Medium-class"></a></li>
+                <li class="social-media-links"><a href="#" class="GitHub-profile"><img src="./Images/github.svg" alt="Github logo" class="Github-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Angellist-profile"><img src="./Images/angellist.svg" alt="Angellist logo" class="Angellist-class"></a></li>
+            </ul>
+        </section>
         <!-- description container ends -->
       </div>
       <!-- Menu and about page start -->
-    <div class="menu-list">
+    <section class="menu-list">
```
Comment (awais-amjed) [inline index.html:49]: Kindly note these issues in the `Recent works` Section 🤗
- _The heading should be on one line_
- _Kindly expand the work cards to match the Figma design as shown below_

Your design:

image

Figma design:

image

File: index.html [hunk 5]
```diff
@@ -8,51 +8,58 @@
...
         <div class="works-heading">
```
Comment (awais-amjed) [inline index.html:50]: Kindly note that the size of button increases too much when you hover on it, kindly fix that 🤗

image

File: index.html [hunk 6]
```diff
@@ -202,23 +209,20 @@ <h3>
...
+                About me
+            </h2>
+            <p class="about-description">
+                I can help you build a product,<br>
+                feature or website Look through<br>
+                some of my work and experience! If<br>
+                you like what you see and have a<br>
+                project you need coded,<br>
+                don’t hestiate to contact me.
+            </p>
```
Comment (awais-amjed) [inline index.html]: Kindly also fix the design of the about me section to match the Figma design as shown below 🤗
- _Make the section flexible to fill the screen_
- _Please add the background image to the section_

Your design:

image

Figma design:

image

File: index.html [hunk 7]
```diff
@@ -270,15 +274,13 @@ <h3>
...
+                <ul class="list-about">
+                    <li class="list-tag">Codekit</li>
+                    <li class="list-tag">GitHub</li>
+                    <li class="list-tag">Codepen</li>
+                    <li class="list-tag">Gitlab</li>
+                    <li class="list-tag">Terminal</li>
+                </ul>
             </div>
         </div>
         <footer class="contact-footer">
```
Comment (awais-amjed) [inline index.html]: Kindly fix the contact section's style to match the Figma design too 🤗

Your design:

image

Figma design:

image

File: README.md [hunk 8]
```diff
@@ -1,8 +1,10 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Mobile_Project
+# Desktop_version
```
Comment (awais-amjed) [inline README.md:3]: [OPTIONAL] It's better to use a name that described your whole project e.g. `My Portfolio` or `Portfolio` 🤗

File: README.md [hunk 9]
```diff
@@ -1,8 +1,10 @@
...
+
+
+> This is our second project that includes more in-depth HTML and CSS. The task involves coding a figma design which mimics our personal project portfolio which we will later be using to present to future employers. The project was added locally (files and linters also set up) and all changes were successful as linters errors were all fixed. The introductory, "my recent works", "about me" and the contact form page has been finished for the mobile version and desktop version. Media queries were added in order to make the design responsive and animations were also added to certain buttons.
```
Comment (awais-amjed) [inline README.md]: [OPTIONAL] Please keep the description to 1-2 lines 😃

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @keebot-png , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [ ] Kindly implement button interaction as shown in the Figma design 🤗

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 10]
```diff
@@ -882,32 +880,31 @@ textarea { resize: none; }
...
 
   .contact-container {
     display: flex;
-    justify-content: space-around;
-    border: 2px solid red;
-    width: 70%;
+    flex-shrink: 1, 5;
+    justify-content: center;
+    margin-right: 5rem;
   }
```
Comment (levy002) [inline style.css]: - [ ] As requested by the previous reviewer, kindly make sure to align correctly the contact section content correctly as on the provided Figma design. :wink: 

    - ### Your design
   image


    - ### Figma design
  
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @keebot-png   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses:  on how you addressed most of the requested changes :+1: 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 10 above)
Comment (keebot-png) [reply style.css]: Hi @levy002 Please help me understand what it means by the "correct" alignment. The previous code reviewer was specific in his request with regard to the contact form alignment, which has been corrected from column to row using flexbox. The task is to code the portfolio project "as close as possible to your chosen template" and so therefore it implies that it does not have to be 100% identical. A lot of time went into correcting such changes and I found it extremely difficult to replicate the desktop contact form (especially in terms of centering it) and tried my best to code it as close as possible all while fitting the different screen sizes. You are not providing me with any specific direction and are vague in your request. I kindly ask if you may give me specific instructions or consider a re-review.

Comment (levy002) [reply style.css]: Hello @keebot-png, excellent for communicating well and politely. While I was testing your website I noticed that the `input` and `textarea` in your form are taking a long width than the `form` container itself. So to be more clear on this point kindly adjust the width of your `input` and even your `textarea` (message part). If you check it again you have given a long width to those form elements. I highly recommend you adjust it again in order to make it responsive. I have checked your website desktop version from `768px` to `1440px` breakpoints but this contact form container fits correctly the screen when the width is `1650px` which is out of the required range.

Comment (levy002) [reply style.css]: If u still feel stuck kindly reach out on slack so we can try to solve this issue together! Thanks :smiley:

**Review by danmainah — APPROVED** [review]
Hi @keebot-png  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**